commercetools/provider.go

```diff
@@ -17,52 +18,57 @@ func Provider() terraform.ResourceProvider {
 	return &schema.Provider{
 		Schema: map[string]*schema.Schema{
 			"client_id": {
-				Type:     schema.TypeString,
-				Optional: true,
-				DefaultFunc: schema.MultiEnvDefaultFunc([]string{
```

Comment:
I couldn't find a reason for using `MultiEnvDefaultFunc`, so I opted to use `EnvDefaultFunc` (same for the other fields)

Reply:
Makes sense :+1: